<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>helpwindow</class>
 <widget class="QDialog" name="helpwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>754</width>
    <height>631</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>754</width>
    <height>631</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(181, 181, 181);</string>
  </property>
  <widget class="QPushButton" name="Minus">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>530</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>--</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Plus">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>530</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QTreeWidget" name="treeWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>201</width>
     <height>611</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>201</width>
     <height>611</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>201</width>
     <height>611</height>
    </size>
   </property>
   <column>
    <property name="text">
     <string/>
    </property>
   </column>
   <item>
    <property name="text">
     <string>Unos:</string>
    </property>
    <item>
     <property name="text">
      <string>Unos Knjiga</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Unos Učenika</string>
     </property>
    </item>
   </item>
   <item>
    <property name="text">
     <string>Izmjena:</string>
    </property>
    <item>
     <property name="text">
      <string>Izmjena Knjiga</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Izmjena Učenika</string>
     </property>
    </item>
   </item>
   <item>
    <property name="text">
     <string>Personalizacija:</string>
    </property>
    <item>
     <property name="text">
      <string>Providnost i Boje</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Veličina Fonta</string>
     </property>
    </item>
   </item>
   <item>
    <property name="text">
     <string>Brisanje</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Pregled razreda i knjiga</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Pregled Loga</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Podizanje Knjige</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Vraćanje Knjige</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Kreiranje Razreda</string>
    </property>
   </item>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>217</x>
     <y>403</y>
     <width>521</width>
     <height>151</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>521</width>
     <height>151</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>521</width>
     <height>151</height>
    </size>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>217</x>
     <y>560</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>131</width>
     <height>61</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>131</width>
     <height>61</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>217</x>
     <y>10</y>
     <width>521</width>
     <height>391</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>521</width>
     <height>261</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>521</width>
     <height>411</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>613</x>
     <y>560</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>131</width>
     <height>61</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>131</width>
     <height>61</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>354</x>
     <y>560</y>
     <width>253</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>253</width>
     <height>61</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>253</width>
     <height>61</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>treeWidget</zorder>
  <zorder>textEdit</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>label</zorder>
  <zorder>pushButton</zorder>
  <zorder>label_2</zorder>
  <zorder>Plus</zorder>
  <zorder>Minus</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
